build/quantities.js

```
@@ -529,9 +531,25 @@ SOFTWARE.
         for (var j = 0; j < definition[0].length; j++) {
           UNIT_MAP[definition[0][j]] = unitDef;
         }
+        if (isBase) {
+          if (BASE_UNITS.indexOf(unitDef) === -1) {
+            BASE_UNITS.push(unitDef);
+          }
```

else throw error? silent failures are difficult to debug.